recipes/belt/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "BELT" %}
+{% set version = "BELTv1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qianglbl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: cbfe5081595ba91bb88e4cb9deec001f60c73571a8cc10788ac60d789089f843
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  script: 
+    - ln -sf $BUILD_PREFIX/bin/$CONDA_TOOLCHAIN_HOST-gfortran $BUILD_PREFIX/bin/gfortran
+    - cd src
+    - make -f Makefile
+    - mkdir -p $PREFIX/bin
+    - cp xeblt $PREFIX/bin
+    - cp -r ../examples/example1 $PREFIX/test
+
+requirements:
+  build:
+    - make
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+
+test:
+  commands:
+    - which xeblt
+    - cd $PREFIX/test
+    - $PREFIX/bin/xeblt
+
+about:
+  home: https://github.com/qianglbl/BELT
+  summary: 'BELT: BEam Longitudinal Tracking (forward or backward)'
+  description: |
+              This is a serial/parallel code for fast electron beam longitudinal 
+              forward or backward tracking through an electron linear accelerator. 
+              It includes longitudinal space-charge effect, structure and 
+              resistive wall wakefields, longitudinal coherent synchrotron 
+              radiation effect.
+              Copyright Notice
+              BEam Longitudinal Tracking (BELT) Copyright (c) 2024, 
+              The Regents of the University of California, 
+              through Lawrence Berkeley National Laboratory (subject to receipt of 
+              any required approvals from the U.S. Dept. of Ener gy). All rights reserved.
+  license: BSD-4-Clause-UC
```

The license actually looks like this one
```suggestion
  license: BSD-3-Clause-LBNL
```